docs/snippets/vue/button-story-click-handler-args.vue3.js.mdx

```diff
@@ -0,0 +1,18 @@
+```js
+// Button.stories.js
+
+export const Text = (args) => ({
+  components: { Button },
+  setup() {
+    return {
+      label: args.label,
+      onClick: action('clicked'),
+    };
+  },
+  template: '<Button @onClick="onClick" :label="label" />',
+});
+Text.args = {
+  label: 'Hello',
+  onClick: action('clicked'),
```

Comment:
Is this needed it we have it inside the `setup()`?
```suggestion
```

Reply:
We could drop it. Fixed with the latest commits.